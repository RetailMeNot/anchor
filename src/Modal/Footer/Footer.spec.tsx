// REACT
import * as React from 'react';
import * as renderer from 'react-test-renderer';
// COMPONENT
import { Modal } from '..';
const { Footer } = Modal;

describe('Component: Modal.Footer', () => {
    it('should render', () => {
        const subject = <Footer>Text</Footer>;
        const tree = renderer.create(subject).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
