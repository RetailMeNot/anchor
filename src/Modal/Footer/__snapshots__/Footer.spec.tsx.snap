// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Modal.Footer should render 1`] = `
.c0 {
  box-sizing: border-box;
  width: 100%;
  margin-top: auto;
  border-bottom-left-radius: inherit;
  border-bottom-right-radius: inherit;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-order: 1;
  -ms-flex-order: 1;
  order: 1;
  justify-self: flex-end;
}

<div
  className="anchor-modal-footer c0"
>
  Text
</div>
`;
